README.md

```diff
@@ -347,6 +347,7 @@ HEROKU_JANIS_APP_NAME=janis-staging ./scripts/serve-local.sh
 ```
 
 #### Syncing prod data to staging
-1. Create a new seeding data backup sourced from prod, using `migration_test.sh`
-2. Drop the staging database (go to heroku and delete the database on staging) 
+
+1. Create a new seeding data backup sourced from prod, using `LOAD_PROD_DATA=on migration_test.sh`
```

Comment:
note: when created from prod data we see that it was sourced from prod in the [seeding_datadump_metadata.txt](https://github.com/cityofaustin/joplin/blob/f847c0b06947c4d4a04160573ff10a429755e9f7/joplin/db/system-generated/seeding_datadump_metadata.txt#L7)